### PR TITLE
Adding address.js

### PR DESCRIPTION
**Pull request by Jcrpelaez**

### CODE REVIEW

File: assets/js/address.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+var settings = {
+	"async": true,
+	"crossDomain": true,
+	"url": "https://realtymole-rental-estimate-v1.p.rapidapi.com/rentalPrice?address=" + address + "&bedrooms=" + bedrooms + "&bathrooms=" + bathrooms + "&propertyType=" + propertyType + "&squareFootage=" + squareFootage + "&compCount=" + results,
+	"method": "GET",
+	"headers": {
+		"x-rapidapi-host": "realtymole-rental-estimate-v1.p.rapidapi.com",
+		"x-rapidapi-key": "c403db74fcmsh82c18191678d823p1aeb6ajsneba4140264a8"
+	}
+}
```
Comment (Jcrpelaez) [inline assets/js/address.js:17]: Awesome use of the api key!

**Review by Jcrpelaez — COMMENTED** [review]
Looks Good!

**Review by jmike2689 — COMMENTED** [review]
Thanks for the kind words - JM

**Review by Jcrpelaez — COMMENTED** [review]
Looks good!

**Review by jmike2689 — COMMENTED** [review]
Thanks!

**Review by jmike2689 — APPROVED** [review]
Approving

**Pull request merged**